pages/faq.md

```diff
@@ -113,6 +113,17 @@ git clone https://github.com/dogi/ole--vagrant-vi.git
 cd ole--vagrant-vi
 vagrant up
 ```
+## 'First Steps' Software Summary
+
+###### _The aim of the ‘First Steps’ is to introduce prospective interns to the software that they will be using, and make sure they are familiar with each. While each step goes into detail on the specific program(s) at hand, it can be easy at times to lose sight of the bigger picture. To that end, below is a brief synopsis of the primary tools you will be using/learning about in the first steps, and how they work together to empower our collaborative development environment._
```

Comment:
If you want to highlight, a note-like formatting or quote is better than Header 6 I think.